python/pyface3d/pyface3d.cxx

```diff
@@ -963,7 +963,7 @@ PYBIND11_MODULE(face3d, m)
 #endif
 
   py::class_<face3d::vertex_localizer>(m, "vertex_localizer")
-    .def(py::init<face3d::triangle_mesh const&>())
+    .def(py::init<face3d::triangle_mesh const&, int>(), py::arg("semantic_map"), py::arg("cuda_device")=0)
```

Comment:
the first argument should be the mesh whose vertices you're going to be localizing, not the semantic map.  (The semantic_map gets passed into the __call__ function)

Reply:
fixed